utils/package.json

```diff
@@ -5,23 +5,23 @@
   "author": "NewRelic",
   "license": "Apache-2.0",
   "scripts": {
-    "validate-quickstart-install-plans": "node validate-quickstart-install-plans.js",
-    "create-validate-install-plans": "node create-validate-install-plans.js",
-    "create-validate-pr-quickstarts": "node create_validate_pr_quickstarts.js",
+    "validate-quickstart-install-plans": "ts-node validate-quickstart-install-plans.js",
+    "create-validate-install-plans": "ts-node create-validate-install-plans.ts",
+    "create-validate-pr-quickstarts": "ts-node create_validate_pr_quickstarts.ts",
     "check-dashboard-name-uniqueness": "ts-node check_dashboard_name_uniqueness.ts",
     "test": "jest -i",
-    "check-quickstart-uniqueness": "node check_quickstart_uniqueness.js",
-    "validate-images": "node validate_images.js",
+    "check-quickstart-uniqueness": "ts-node check_quickstart_uniqueness.js",
+    "validate-images": "ts-node validate_images.js",
     "format": "prettier --write \"./**/*.js\"",
-    "validate-icons": "node validate_icons.js",
-    "generate-uuids": "node generate-uuids.js",
-    "sanitize-dashboard": "node sanitize_dashboards.js",
-    "generate-schema-docs": "node graphql-schemas/generate-schema-docs.js"
+    "validate-icons": "ts-node validate_icons.js",
+    "generate-uuids": "ts-node generate-uuids.js",
+    "sanitize-dashboard": "ts-node sanitize_dashboards.js",
+    "generate-schema-docs": "ts-node graphql-schemas/generate-schema-docs.js"
   },
   "dependencies": {
     "@actions/core": "^1.4.0",
     "@types/glob": "^7.2.0",
-    "@types/node": "^17.0.23",
+    "@types/node": "^17.0.24",
```

Comment:
It looks like we have _some_ types in `devDependencies` and _some_ in `dependencies`. Because we don't actually bundle this stuff for production, we can probably just put all of these in dependencies (or dev dependencies...your call).

Reply:
moved into dependencies